api/nvidia/v1/clusterpolicy_types.go

```diff
@@ -148,6 +148,9 @@ type OperatorSpec struct {
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="On OpenShift, enable DriverToolkit image to build and install driver modules"
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:booleanSwitch"
 	UseOpenShiftDriverToolkit *bool `json:"use_ocp_driver_toolkit,omitempty"`
+
+	// UseDevicePluginCDIDevicesFeature indicates if the device plug-in should be configured to use the CDI devices feature
```

Comment:
One question on the UX for this: Should this be under the `cdi` object in the cluster policy?

Reply:
It's an option that only affects the operator, but is related to CDI. I don't know to which it binds more strongly. So I picked one. It's easy enough to change that in the PR, but of course hard after. Maybe other folks can chime in. Ultimately I will defer to you and your team.